datafusion/core/src/physical_optimizer/parallel_sort.rs

```diff
@@ -1,93 +0,0 @@
-// Licensed to the Apache Software Foundation (ASF) under one
-// or more contributor license agreements.  See the NOTICE file
-// distributed with this work for additional information
-// regarding copyright ownership.  The ASF licenses this file
-// to you under the Apache License, Version 2.0 (the
-// "License"); you may not use this file except in compliance
```

Comment:
Moved this to planner - seems a bit more simple. We don't need access to the parent anymore now.